builtin/mainmenu/textures.lua

```diff
@@ -145,6 +145,17 @@ function mm_texture.set_game(identifier,gamedetails)
 	
 	local path = gamedetails.path .. DIR_DELIM .."menu" ..
 									 DIR_DELIM .. identifier .. ".png"
+	if core.setting_getbool("random_menu_images") == true then
+		local count = gamedetails.image_count
+		if tonumber(count) == nil then
```

Comment:
It would generally make way more sense to pass the image count around as integer in the C++ and Lua code instead of using a string all the way until this code.